Cargo.toml

```diff
@@ -8,3 +8,11 @@ members = [
     "crates/apollo-smith",
     "fuzz",
 ]
+
+default-members = [
```

Comment:
temporarily removing fuzz from workspace members for CI. not a good idea to merge it like this though.

Reply:
(router-bridge is extremely fussy and is refusing to build in CI due to too many failures)